ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20-periodics.yaml

```diff
@@ -1,23 +1,24 @@
 periodics:
 - agent: kubernetes
   cluster: build05
-  cron: 0 0 * * *
+  cron: 0 4 * * *
```

Comment:
I'm probably missing something: this doesn't match the cron job's config I've seen above.